lib/src/frontend/format_stack_trace.dart

```diff
@@ -6,10 +6,17 @@ import 'package:stack_trace/stack_trace.dart';
 
 import '../backend/stack_trace_formatter.dart';
 
+/// The default formatter to use for formatting stack traces.
+///
+/// This is used in situations where the zone-scoped formatter is unavailable,
+/// such as when running via `dart path/to/test.dart'.
+final _defaultFormatter = new StackTraceFormatter();
+
 /// Converts [stackTrace] to a [Chain] according to the current test's
 /// configuration.
 ///
 /// If [verbose] is `true`, this doesn't fold out irrelevant stack frames. It
 /// defaults to the current test's `verbose_trace` configuration.
 Chain formatStackTrace(StackTrace stackTrace, {bool verbose}) =>
-    StackTraceFormatter.current.formatStackTrace(stackTrace, verbose: verbose);
+    (StackTraceFormatter.current ?? _defaultFormatter)
```

Comment:
Ahh beat me to it. I meant to do this change as I had to do a similar work around for Bolt.